exercises/binary-search/introduction.md

```diff
@@ -1,7 +1,7 @@
 # Introduction
 
 You have stumbled upon a group of mathematicians who are also singer-songwriters.
-They have written a song for each of their favorite numbers, and, as you can imagine, they have a lot of favorite numbers.
+They have written a song for each of their favorite numbers, and, as you can imagine, they have a lot of favorite numbers (like 0 or 73 or 6174).
```

Comment:
One final thought: would it be nice to link to wikipedia for the favorite numbers as 73 and 6174 do have some meaning. Just a thought, feel free to ignore.

Reply:
Yeah I like that.